example/cumulus-tf/terraform.tfvars.example

```diff
@@ -33,6 +33,12 @@ cmr_password    = "password"
 cmr_provider    = "CUMULUS"
 cmr_username    = "username"
 
+cmr_oauth_provider = "launchpad"
+
+launchpad_api = "launchpadApi"
+launchpad_certificate = "certificate"
+launchpad_passphrase = "passphrase"
```

Comment:
Can we add a note that these are optional if you want to use launchpad? Since most of us haven't been deploying/testing with CMR launchpad integration